compiler/tests/testSuite.bal

```diff
@@ -57,15 +68,41 @@ function testCompileEU(string path, string kind) returns file:Error|io:Error? {
     }
 }
 
-function compileErrorLineNumber(CompileError err) returns int? {
-    if err is io:Error {
-        return ();
+function moduleDir(string filePath) returns string|file:Error? {
+    string subModPath = filePath.substring(0, filePath.length()-4) + ".modules";
+    if check file:test(subModPath, file:IS_DIR) {
+        return check file:normalizePath(subModPath, file:CLEAN);
+    }
+}
+
+function expectedErrorLocation(CompileError err, string path) returns [int, string]|file:Error|io:Error? {
+    string? modulePath = check moduleDir(path);
+    string? errorFileName = ();
+    int? errorLineNumber = ();
+    if modulePath is string {
+        foreach var dir in check file:readDir(modulePath) {
```

Comment:
dir has type MetaData, so use `md` not dir.

Reply:
fixed